src/Client.php

```diff
@@ -77,11 +79,11 @@ class Client
     /** @var SignatureGeneratorInterface */
     protected $signatureGenerator;
 
-    public function __construct(array $config = [])
+    public function __construct(array $config = [], string $environment = 'test')
```

Comment:
Friendly reminder. The default `test` will change the default behavior of client and break backward compatibility.